src/scss/components/_form.scss

```diff
@@ -7,6 +7,37 @@
 **/
 
 .form {
+  position: relative;
+  overflow: hidden;
+
+  .envelop {
+    background: get-color(reward);
+    width: 100%;
+    position: absolute;
+    left: 50%;
+    transform: translate(-50%, 0);
+    padding: 30px 30px 20px 30px;
+
+    &__title {
+      margin: 0;
+      text-align: center;
+      color: get-color(air);
+      text-transform: uppercase;
+      font-size: 14px;
+      font-weight: 900;
+    }
+  }
+
+  .svg-triangle {
+    width: 100%;
```

Comment:
This one is a element of form? **If no**, we can move this to a new element. **If yes**, let's add the `form` scope to this one too.